src/components/Button/style.scss

```diff
@@ -23,7 +23,13 @@
   padding: 0 16px;
   min-width: 36px;
   min-height: 36px;
-  line-height: 36px;
+  line-height: 1.3;
+
+  &:after{
```

Comment:
can u explain to me why we need it?

Reply:
yup, flex in ie 11. 
https://stackoverflow.com/questions/19371626/flexbox-not-centering-vertically-in-ie

Reply:
omg :D Didn't other solutions work?

Reply:
didn't because we cannot set constant btn height  😢